CHANGELOG.md

```diff
@@ -6,7 +6,11 @@
 ### Added
 
 - Publish package to npm using tags
+- Entry node - subscribe for all public channels
```

Comment:
Do all waggle nodes subscribe to all public channels? 

If not, why not? It seems like this would be a good way to improve syncing speeds and reliability once we don't have an entry node (so that more peers will have a copy of your channels).